Automattic/wp-calypso
# Jetpack Cloud: Render "today" in Activity Log

#### Changes proposed in this Pull Request

* Render "Today" in Activity Log for the current date

![Untitled 4](https://user-images.githubusercontent.com/2810519/82096717-51465700-96b6-11ea-93ef-2b646cd42d17.png)


#### Testing instructions

1. navigate to `/backups/activity`
2. verify "Today" appears in place of the current date
